.github/scripts/test-project.sh

```diff
@@ -73,6 +76,16 @@ if [ $code -eq 0 ]; then
   echo "-----------------------------"
 fi
 
+# confirm existence of correct engine
+if [ $code -eq 0 ]; then
+  echo "-------------- Checking Binaries ---------------"
+  bash ../../.github/scripts/check-cli-binaries.sh $dir $project
+  bash ../../.github/scripts/check-client-binaries.sh $dir $project
+  echo "------------------------------------------------"
+fi
+
+# TODO parse output of npx prisma -v --json for correct file/path
```

Comment:
Didn't we already have that along the way?

Reply:
Yes but that is part of an individual test (core-features/napi-preview-feature) which uses ts-node and jest and not a bash script

Reply:
Ah true. Thanks for the reminder.